### PR TITLE
Add support of highlight AMP-compatible themes and plugins in network pages

### LINKED ISSUE #6725: Highlighting AMP-compatible themes and plugins in network theme/plugin screens is not working properly.
### Bug Description

The highlighting theme and plugin are not working properly in the network theme and plugin screens in the MU site setup.

**Installed Themes.**

The AMP-compatible theme does not show the message of "AMP Compatible" as we show on the installed plugin page.
image

**Add New Theme.**

The "AMP Compatible" tab and "AMP Compatible" badges are missing.
image

**Add New Plugin (Populer Tab)**

The "AMP Compatible" badges are missing.
image

**Add New Plugin (AMP Compatible Tab)**

Additional detail section are rendring.
image

### Expected Behaviour

**Installed Themes.**
- Show "AMP Compatible" message beside the theme author and theme site link.

**Add New Theme.**
- Add "AMP Compatible" badge for AMP-compatible themes.
- Add the "AMP Compatible" tab which shows a list of AMP-compatible themes.

**Add New Plugin (Populer Tab)**
- Add "AMP Compatible" badge for AMP-compatible plugins.

**Add New Plugin (AMP Compatible Tab)**
- Remove addition detail section from plugin card.
image

### Screenshots

_No response_

### PHP Version

_No response_

### Plugin Version

2.2-alpha

### AMP plugin template mode

Standard, Transitional, Reader

### WordPress Version

_No response_

### Site Health

_No response_

### Gutenberg Version

_No response_

### OS(s) Affected

_No response_

### Browser(s) Affected

_No response_

### Device(s) Affected

_No response_

### Acceptance Criteria

_No response_

### Implementation Brief

_No response_

### QA Testing Instructions

_No response_

### Demo

_No response_

### Changelog Entry

_No response_

### PR DESCRIPTION
**Pull request by dhaval-parekh**
## Summary

Fixes #6725 
Related #2313 

Installed Themes.
| Before                                                                                                                                    | After                                                                                                                                     |
|-------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
| image | image |

Add New Theme.

| Before                                                                                                                                    | After                                                                                                                                     |
|-------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
| image | image |

Add New Plugin (Populer Tab)
| Before                                                                                                                                    | After                                                                                                                                     |
|-------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
| image | image |

Add New Plugin (AMP Compatible Tab)

| Before                                                                                                                                    | After                                                                                                                                     |
|-------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------|
| image | image |

## Checklist

- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/wiki/Engineering-Guidelines) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: src/Admin/AmpThemes.php [hunk 1]
```diff
@@ -249,4 +254,23 @@ public function filter_themes_api( $override, $action, $args ) {
...
+	 * @param string[] $theme_meta An array of the theme's metadata.
+	 * @param string   $stylesheet Directory name of the theme.
+	 *
+	 * @return string[] An array of the theme's metadata.
+	 */
+	public function filter_theme_row_meta( $theme_meta, $stylesheet ) {
+
+		$amp_themes = wp_list_pluck( $this->get_themes(), 'slug' );
+
+		if ( ! empty( $stylesheet ) && in_array( $stylesheet, $amp_themes, true ) ) {
```
Comment (delawski) [inline src/Admin/AmpThemes.php]: Is `! empty( $stylesheet )` needed at all? It seems that [`WP_Theme`](https://github.com/WordPress/wordpress-develop/blob/5.8.1/src/wp-includes/class-wp-theme.php#L112-L120) will always return a value for the `$stylesheet` property (and it is what we're ultimately receiving in this filter).

**Review by delawski — APPROVED** [review]
The code looks good to me. I also verified that the AMP compatible labels have been added to themes and plugins on a multisite install.

I left one minor comment in the code.

File: src/Admin/AmpThemes.php (hunk 1 above)
Comment (westonruter) [reply src/Admin/AmpThemes.php]: Right, and even if it is empty then the `in_array( $stylesheet, $amp_themes, true )` check will then fail.
```suggestion
		if ( in_array( $stylesheet, $amp_themes, true ) ) {
```

**Review by milindmore22 — APPROVED** [review]
LGTM

**Pull request merged**